<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>httpInEditor</class>
 <widget class="QWidget" name="httpInEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>335</width>
    <height>307</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="1">
    <widget class="QCheckBox" name="httpTransparentCB">
     <property name="text">
      <string>Allow Transparent</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_13">
     <property name="text">
      <string>Timeout</string>
     </property>
     <property name="buddy">
      <cstring>httpTimeoutSpinBox</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSpinBox" name="httpTimeoutSpinBox">
     <property name="suffix">
      <string>s</string>
     </property>
     <property name="maximum">
      <number>999</number>
     </property>
     <property name="value">
      <number>300</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Users</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="httpAddUserTxt">
         <property name="placeholderText">
          <string>Username</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="httpAddPasswordTxt">
         <property name="placeholderText">
          <string>Password</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="httpAddUserBtn">
         <property name="icon">
          <iconset resource="../../../../../resources.new.qrc">
           <normaloff>:/assets/icons/ui_dark/add.svg</normaloff>:/assets/icons/ui_dark/add.svg</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="httpRemoveUserBtn">
         <property name="icon">
          <iconset resource="../../../../../resources.new.qrc">
           <normaloff>:/assets/icons/ui_dark/minus.svg</normaloff>:/assets/icons/ui_dark/minus.svg</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QListWidget" name="httpAccountListBox"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../../../resources.new.qrc"/>
 </resources>
 <connections/>
</ui>
